tests/e2e/utils/PreferencesHandler.ts

```diff
@@ -10,6 +10,10 @@ export class PreferencesHandler {
     constructor(@inject(CLASSES.CheApiRequestHandler) private readonly requestHandler: CheApiRequestHandler) {
     }
 
+    /**
+     * Works properly only if set before workspace creation,
```

Comment:
I think it would be better to change `before workspace creation` to `before workspace startup`, to be more clear about what the exact problem is.